src/utils/time.ts

```diff
@@ -15,7 +15,13 @@ function getContestStatuses(
       contestStatus: Status.UPCOMING,
     };
   }
-  if (isBefore(currentTime, botRaceEnd) && isAfter(currentTime, start)) {
+  if (isAfter(currentTime, end)) {
+    return {
+      botRaceStatus: Status.ENDED,
+      contestStatus: Status.ENDED,
+    };
+  }
+  if (isBefore(currentTime, botRaceEnd) && (isAfter(currentTime, start) || isEqual(currentTime, start))) {
```

Comment:
Added isEqual here, it fixes the issue where if currentTime and start _happen_ to be equivalent the `undefined` status' get returned.